.github/workflows/build.yml

```diff
@@ -0,0 +1,213 @@
+name: Build and Test
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: "*"
+
+jobs:
+  build:
+    name: Build tljh_repo2docker
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Conda environment with Micromamba
```

Comment:
Do we need micromamba, or would `jupyterlab/maintainer-tools/.github/actions/base-setup@v1` be enough?